lib/lhm/migrator.rb

```diff
@@ -205,5 +206,11 @@ def index_ddl(cols, unique = nil, index_name = nil)
     def order_column
       @options[:order_column] || @origin.pk
     end
+
+    def assert_valid_idx_name(index_name)
+      if index_name && !(index_name.is_a?(String) || index_name.is_a?(Symbol))
+        raise ArgumentError, "index_name must be a string or symbol"
```

Comment:
> Do we want to check that in Ruby?

Maybe, you are right about the migration failing.  I can be very annoying when indexes are derived from the column names and you have to wait after deploying to realize shit's broken. On the other hand ppl should test locally.